generators/new-wp/templates/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "moxie/<%= name %>",
+  "require": {
+    "php": ">=5.4",
+    "moxie-leean/wp-headless-theme": "dev-master",
+    "moxie/acf": "dev-master",
```

Comment:
Let me follow up with Dmitry on the legal aspects... as they are locked down so this doesn't allow you to actually access the plugins (without access to bitbucket).